src/features/Applet/Schedule/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,160 @@
+import { useCallback, useMemo, useState } from 'react';
+import {
+  Calendar as ReactCalendar,
+  SlotInfo,
+  momentLocalizer,
+  Culture,
+  DateLocalizer,
+  Formats,
+  View,
+  ToolbarProps,
+  HeaderProps,
+} from 'react-big-calendar';
+import 'react-big-calendar/lib/css/react-big-calendar.css';
+import moment from 'moment';
+import 'moment/locale/fr';
+
+import i18n from 'i18n';
+import { Svg } from 'components';
+import { variables } from 'styles/variables';
+
+import { CreateActivityPopup } from '../CreateActivityPopup';
+import { EditActivityPopup } from '../EditActivityPopup';
+import { RemoveScheduledEventPopup } from '../RemoveScheduledEventPopup';
+import { Toolbar } from './Toolbar';
+import { MonthEvent } from './MonthEvent';
+import { YearView } from './YearView';
+import { MonthView } from './MonthView';
+import { MonthHeader } from './MonthHeader';
+import { mockedEvents } from './Calendar.const';
+import { StyledCalendarWrapper, StyledAddBtn } from './Calendar.styles';
+import { CalendarEvent, CalendarView } from './Calendar.types';
+import { getEventsWithOffRange } from './Calendar.utils';
+
+moment.updateLocale('en-gb', {
+  week: {
+    dow: 1,
+  },
+});
+
+const momentLocalize = momentLocalizer(moment);
+
+export const Calendar = () => {
+  const { t } = i18n;
+  const [activeView, setActiveView] = useState<CalendarView>(CalendarView.month);
+  const [date, setDate] = useState<Date>(new Date());
+  const [createActivityPopupVisible, setCreateActivityPopupVisible] = useState(false);
+  const [editActivityPopupVisible, setEditActivityPopupVisible] = useState(false);
+  const [removeScheduledEventPopupVisible, setRemoveScheduledEventPopupVisible] = useState(false);
+
+  const events = useMemo(() => getEventsWithOffRange(mockedEvents, date), [date]);
```

Comment:
Are you sure about useMemo and useCallback hooks here?

Reply:
Why do you think they don't fit here?